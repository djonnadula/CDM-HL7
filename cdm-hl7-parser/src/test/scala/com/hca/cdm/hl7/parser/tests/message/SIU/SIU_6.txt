MSH|^~\&|COCQV|COCQV|OV|OV|201702170812||SIU^S18|MT_COCQV_SIU_SCHQV.170217.1|T|2.5||SCA^00104||
PID|1||F000670972^^^COCQV^MRN|F560134|GARZA^CHRISTIANO^PASQUELA||19440622|M||||||||||F01036950890 
PV1|1|O|F.CATHA^F.2102^A^COCQV|EL|||ARGAS^Ramazona^Chartruese|||||||||||SDC|||||||||||||||||||||||REG 
SCH|340192||E|NEW|ARGAS^Ramazona^Chartruese|CAR|COCQV|CATH LAB 1^CATH LAB 1|ARGAS^Ramazona^Chartruese||201702170642|201702170642^FCL.MG|201702170642 
ZTM|PT ARRIVES OPS|PT READY|||PT ARRIVES HOLDING|PT READY|||ANESTHESIA START|PT IN ROOM^201702170719|SURGEON IN ROOM|PROCEDURE START|PROCEDURE END|SURGEON LEAVES ROOM|PT LEAVES ROOM^201702170811||PT ARRIVES PACU|ANESTHESIA END|PT DISCHARGE PHASE I|PT PACU HOLDING OUT|PT ARRIVES PHASE 2|PT DISCHARGE PHASE 2 
ZCP|201702170811|FCL.MG 
ZPO|LOCAL^LOCAL ANESTHESIA||CO^Completed 
ZAO|1|F.CHAN^CATH HEART W/VENT/ANGIOGRAPHY^CATH HEART W/VENT/ANGIOGRAPHY|P||CATH^CATH LAB PROCEDURES|ARGAS^Ramazona^Chartruese|CL 
ZPP|1|F.CHAN^CATH HEART W/VENT/ANGIOGRAPHY^CATH HEART W/VENT/ANGIOGRAPHY|P||CATH^CATH LAB PROCEDURES|ARGAS^Ramazona^Chartruese 
ZST|1|STFANT^Anesthesiologist 
ZIT|1|CATH LAB^CATH LAB|CATH LAB^CATH LAB|017361^^PATCH HMSTC SLVR D-ST TOP DRY^DSTAT|1
ZIT|2|CATH LAB^CATH LAB|CATH LAB^CATH LAB|076288^^TRANSDUCER PRSS TRUWV 1 BOND 1^CATH TRANSDUCER PX600|1 