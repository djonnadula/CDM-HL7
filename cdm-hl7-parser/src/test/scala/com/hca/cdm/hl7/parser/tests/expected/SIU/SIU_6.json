{
  "0000.COMN": {
    "sending_facility": "COCQV",
    "message_control_id": "MT_COCQV_SIU_SCHQV.170217.1",
    "medical_record_num": "F000670972",
    "medical_record_urn": "F560134",
    "patient_account_num": "F01036950890 "
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQV",
    "sending_facility": "COCQV",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "201702170812",
    "security": null,
    "message_type": {
      "message_code": "SIU",
      "trigger_event": "S18"
    },
    "message_control_id": "MT_COCQV_SIU_SCHQV.170217.1",
    "processing_id": "T",
    "version_id": "2.5",
    "sequence_num": null,
    "unknown": "SCA^00104"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": {
      "id_num": "F000670972",
      "id_check_digit": null,
      "check_digit_scheme": null,
      "assigning_authority": "COCQV",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "F560134",
    "patient_name": {
      "family_name": "GARZA",
      "given_name": "CHRISTIANO",
      "second_and_further_given_names_or_initials_thereof": "PASQUELA"
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19440622",
    "administrative_sex": "M",
    "patient_alias": null,
    "race": null,
    "patient_address": null,
    "county_code": null,
    "phone_num_home": null,
    "phone_num_business": null,
    "primary_language": null,
    "marital_status": null,
    "religion": null,
    "patient_account_num": "F01036950890 "
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "O",
    "assigned_patient_location": {
      "point_of_care": "F.CATHA",
      "room": "F.2102",
      "bed": "A",
      "facility": "COCQV"
    },
    "admission_type": "EL",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "ARGAS",
      "family_name": "Ramazona",
      "given_name": "Chartruese"
    },
    "referring_doctor": null,
    "primary_care_doctor": null,
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": null,
    "mode_of_arrival": null,
    "vip_indicator": null,
    "admitting_doctor": null,
    "patient_type": "SDC",
    "visit_num": null,
    "financial_class": null,
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": null,
    "reason_for_visit": null,
    "account_status": "REG "
  },
  "0004.SCH": {
    "placer_appointment_id": "340192",
    "filler_appointment_id": null,
    "occurrence_num": "E",
    "placer_group_num": "NEW",
    "schedule_id": {
      "id": "ARGAS",
      "text": "Ramazona",
      "name_of_coding_system": "Chartruese"
    },
    "event_reason": "CAR",
    "appointment_reason": "COCQV",
    "appointment_type": {
      "id": "CATH LAB 1",
      "text": "CATH LAB 1"
    },
    "unknown": "ARGAS^Ramazona^Chartruese",
    "appointment_duration_units": null,
    "appointment_timing_quantity": "201702170642",
    "placer_contact_person": {
      "person_id": "201702170642",
      "family_name": "FCL.MG"
    },
    "placer_contact_phone_num": "201702170642 "
  },
  "0005.ZTM": {
    "unknown": "PT ARRIVES OPS^PT READY^^^PT ARRIVES HOLDING^PT READY^^^ANESTHESIA START^PT IN ROOM^201702170719^SURGEON IN ROOM^PROCEDURE START^PROCEDURE END^SURGEON LEAVES ROOM^PT LEAVES ROOM^201702170811^^PT ARRIVES PACU^ANESTHESIA END^PT DISCHARGE PHASE I^PT PACU HOLDING OUT^PT ARRIVES PHASE 2^PT DISCHARGE PHASE 2 "
  },
  "0006.ZCP": {
    "unknown": "201702170811^FCL.MG "
  },
  "0007.ZPO": {
    "unknown": "LOCAL^LOCAL ANESTHESIA^^CO^Completed "
  },
  "0008.ZAO": {
    "unknown": "1^F.CHAN^CATH HEART W/VENT/ANGIOGRAPHY^CATH HEART W/VENT/ANGIOGRAPHY^P^^CATH^CATH LAB PROCEDURES^ARGAS^Ramazona^Chartruese^CL "
  },
  "0009.ZPP": {
    "unknown": "1^F.CHAN^CATH HEART W/VENT/ANGIOGRAPHY^CATH HEART W/VENT/ANGIOGRAPHY^P^^CATH^CATH LAB PROCEDURES^ARGAS^Ramazona^Chartruese "
  },
  "0010.ZST": {
    "unknown": "1^STFANT^Anesthesiologist "
  },
  "0011.ZIT": {
    "unknown": "1^CATH LAB^CATH LAB^CATH LAB^CATH LAB^017361^^PATCH HMSTC SLVR D-ST TOP DRY^DSTAT^1"
  },
  "0012.ZIT": {
    "unknown": "2^CATH LAB^CATH LAB^CATH LAB^CATH LAB^076288^^TRANSDUCER PRSS TRUWV 1 BOND 1^CATH TRANSDUCER PX600^1 "
  }
}